TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift

```diff
@@ -25,26 +25,25 @@ public protocol FactorPayload {
   var friendlyName: String { get }
   ///The unique SID identifier of the Service.
   var serviceSid: String { get }
-  ///Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
-  ///because the systems that will process this attribute assume it is not directly identifying information.
+  ///Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information) because the systems that will process this attribute assume it is not directly identifying information.
   var identity: String { get }
   ///Type of the factor.
   var factorType: FactorType { get }
 }
 
-///Describes the information required to create a **Factor** which type is **Push**.
+/// Describes the information required to create a **Factor** which type is **Push**.
 public struct PushFactorPayload: FactorPayload {
-  ///A human readable description of this resource, up to 64 characters. For a push factor, this can be the device's name.
+  /// A human readable description of this resource, up to 64 characters. For a push factor, this can be the device's name.
   public let friendlyName: String
-  ///The unique SID identifier of the Service.
+  /// The unique SID identifier of the Service.
   public let serviceSid: String
-  ///Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
+  /// Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
   ///because the systems that will process this attribute assume it is not directly identifying information.
   public let identity: String
   ///Type of the factor, push by default.
   public let factorType: FactorType = .push
-  ///Registration token generated by APNS when registering for remote notifications.
-  public let pushToken: String
+  ///(Optional) Registration token generated by the FCM SDK on initial startup of your app. Sending null or empty will disable sending push notifications for challenges associated to this factor.
```

Comment:
Please keep "...by APNS when registering for remote notifications." and just add the "Sending null or empty..."

Reply:
And we should replace null by nil, right?

Reply:
Sounds good! I just did it on the last pushed commit